packages/compass-aggregations/src/modules/aggregation.ts

```diff
@@ -27,6 +27,9 @@ import { runPipelineConfirmationDescription } from '../utils/modal-descriptions'
 import type { MongoDBInstance } from 'mongodb-instance-model';
 import type { DataService } from '../modules/data-service';
 import toNS from 'mongodb-ns';
+import { createTrack } from '@mongodb-js/compass-telemetry';
+
+const track = createTrack();
```

Comment:
This one should still be using service injection through the plugin, right?

Reply:
thanks!